mne/preprocessing/eyetracking/calibration.py

```diff
@@ -0,0 +1,346 @@
+        data was collected with. For example ``(.531, .298)`` for a monitor with
+        a display area of 531 x 298 mm.
+    screen_distance : float
+        The distance (in meters) from the participant's eyes to the screen.
+    screen_resolution : array-like of shape ``(2,)``
+        The resolution (in pixels) of the screen that the eyetracking data
+        was collected with. For example, ``(1920, 1080)`` for a 1920x1080
+        resolution display.
+
+    Attributes
+    ----------
+    onset : float
+        The onset of the calibration in seconds. If the calibration was
+        performed before the recording started, the onset will be ``0`` seconds.
+    model : str
+        A string, which is the model of the calibration that was applied. For
+        example ``'H3'`` for a horizontal only 3-point calibration, or ``'HV3'`` for a
+        horizontal and vertical 3-point calibration.
+    eye : str
+        The eye that was calibrated. For example, ``'left'``, or ``'right'``.
+    avg_error : float
+        The average error in degrees between the calibration points and the actual gaze
+        position.
+    max_error : float
+        The maximum error in degrees that occurred between the calibration points and
+        the actual gaze position.
+    points : ndarray
+        a 1D structured numpy array of shape ``(n_calibration_points,)``, which contains
+        the data for each calibration point.
+    screen_size : array-like
+        The width and height (in meters) of the screen that the eyetracking data was
+        collected  with. For example ``(.531, .298)`` for a monitor with a display area
+        of 531 x 298 mm.
+    screen_distance : float
+        The distance (in meters) from the participant's eyes to the screen.
+    screen_resolution : array-like
+        The resolution (in pixels) of the screen that the eyetracking data was
+        collected with. For example, ``(1920, 1080)`` for a 1920x1080 resolution
+        display.
+
+    Examples
+    --------
+    Below is an example of data that can be passed to the points parameter:
+    ``>>> data = [(960., 540., 0.23, 950.1,  544.1), (960., 92., 0.38, 967.8, 76. )]``
+    """
+
+    def __init__(
+        self,
+        onset=None,
+        model=None,
+        eye=None,
+        avg_error=None,
+        max_error=None,
+        points=None,
+        screen_size=None,
+        screen_distance=None,
+        screen_resolution=None,
+    ):
+        super().__init__()
+        self["onset"] = onset
+        self["model"] = model
+        self["eye"] = eye
+        self["avg_error"] = avg_error
+        self["max_error"] = max_error
+        if points is not None:
+            self.set_calibration_array(points)
+        else:
+            self["points"] = points
+        self["screen_size"] = screen_size
+        self["screen_distance"] = screen_distance
+        self["screen_resolution"] = screen_resolution
```

Comment:
Since we have to call the super init anyway we might as well have it do some work for us
```suggestion
        super().__init__(
            onset=onset,
            model=model,
            eye=eye,
            avg_error=avg_error,
            max_error=max_error,
            screen_size=screen_size,
            screen_distance=screen_distance,
            screen_resolution=screen_resolution,
        )
        if points is not None:
            self.set_calibration_array(points)
        else:
            self["points"] = points
```

Reply:
Oh nice, thanks for the tip!